test/acceptance/features/http.feature

```diff
@@ -43,3 +43,31 @@ Feature: HTTP client
           | json.inactive         | [FALSE]                   |
           | json.empty            | [EMPTY]                   |
           | json.null             | [NULL]                    |
+  
+  Scenario: Send a POST request defined by a json string
+    Given the HTTP endpoint "[CONF:url]/anything"
+      And the HTTP path "/test3"
+      And the HTTP request headers
+          | Authorization | Bearer access-token |
```

Comment:
You haven't written the Content-Type HTTP header. See my previous comment to make it implicit with the JSON body step.

Reply:
Done, added the implicit indication to clarify